src/object/lit_object.cpp

```diff
@@ -114,4 +110,14 @@ LitObject::set_light_action(const std::string& action)
   m_light_sprite->set_action(action);
 }
 
+
+void
+LitObject::register_class(ssq::VM& vm)
+{
+  ssq::Class cls = vm.addAbstractClass<LitObject>("LitObject", vm.findClass("MovingObject"));
```

Comment:
Is there a reason this inherits MovingObject and not MovingSprite? At the moment you can only change the light sprite through scripting and not the actual sprite.